framework/src/controller/channels/base/constants.js

```diff
@@ -4,6 +4,9 @@ const INTERNAL_EVENTS = Object.freeze([
 	'loading:finished',
 ]);
 
+const eventWithModuleNameReg = /[A-Za-z]+[A-Za-z0-9]*:[A-Za-z]+[A-Za-z0-9]*/;
```

Comment:
no nesting more than 2 is allowed?

Reply:
@MaciejBaj other regex is used: `/^([a-zA-Z][a-zA-Z0-9]*)((?::[a-zA-Z][a-zA-Z0-9]*)+)$/`
If you use this one, app starts again.

Reply:
True, fixed both - nesting multiple actions is now possible: https://regex101.com/r/etJocJ/3